thebeat/stats.py

```diff
@@ -637,17 +650,23 @@ def fft_values(
 
     # Make a sequence of ones and zeroes
     timeseries = thebeat.helpers.sequence_to_binary(sequence, resolution=step_size)
-    duration = sequence.duration
+    timeseries = timeseries if sequence.end_with_interval is True else timeseries[:-1]
+
+    # Remove DC component if necessary
+    if remove_dc:
```

Comment:
Can we get a test that this makes a difference? And also test (possibly in the same test) whether the DC component is indeed equal to 0, then?